src/backend/distributed/utils/reference_table_utils.c

```diff
@@ -250,3 +318,32 @@ CreateReferenceTableColocationId()
 
 	return colocationId;
 }
+
+
+/*
+ * ReferenceTableList function scans pg_dist_partition to create a list of all reference
+ * tables. To create the list, it performs index scan.
```

Comment:
Let's update the function comment. But, also mention that we could use indexScan to increase performance if needed.